src/persist/PersistStructs.hpp

```diff
@@ -0,0 +1,279 @@
+#ifndef PERSIST_STRUCTS_HPP
```

Comment:
Why do we need this extra file? Could they be merged into `Recoverable.hpp`?

Reply:
We need this to be included by `EpochSys.hpp`. `Recoverable.hpp` is at the end of the dependency chain.